Realm/RLMObject.h

```diff
@@ -311,6 +312,17 @@ RLM_ASSUME_NONNULL_BEGIN
  */
 + (nullable instancetype)objectForPrimaryKey:(nullable id)primaryKey;
 
+/**
+ Get objects whose location is within the given bounding box.
+
+ @param box                The bounding box to find objects within.
+ @param latitudeProperty   The property of the object to interpret as the latitude.
+ @param longitudeProperty  The property of the object to interpret as the longitude.
+
+ @return                   An RLMResults of objects of the subclass type in the default Realm whose locations are within the bounding box.
+ */
++ (RLMResults *)objectsWithinBoundingBox:(RLMBoundingBox)box latitudeProperty:(NSString *)latitudeProperty longitudeProperty:(NSString *)longitudeProperty;
```

Comment:
I personally don't think these methods are necessary -- should be sufficient to support querying against `allObjects`